### PR TITLE
add OSA 3.11 build config

### PR DESCRIPTION
**Pull request by mjudeikis**
This will enable us to build 3.11 Images for OSA

/cc kargakis charlesakalugwu 
ptal if I didnt missed anything. It still needs ansible PR to go in

### CODE REVIEW

File: ci-operator/templates/openshift/openshift-azure/build-node-image-3.10.yaml [hunk 1]
```diff
@@ -0,0 +1,351 @@
+# This template is being used for testing OpenShift Ansible image builds for Azure.
```
Comment (kargakis) [inline ci-operator/templates/openshift/openshift-azure/build-node-image-3.10.yaml]: I was hoping we can reuse the existing template. Why is this necessary?

Comment (mjudeikis) [reply ci-operator/templates/openshift/openshift-azure/build-node-image-3.10.yaml]: Hm.. You might be right, let me check

**Comment by mjudeikis** [comment]
I didnt tested it, but it should work @kargakis
